ast/expressions.go

```diff
@@ -700,6 +850,12 @@ type UnaryOperationExpr struct {
 	V ExprNode
 }
 
+// Format the ExprNode into a Writer.
+func (n *UnaryOperationExpr) Format(w io.Writer) {
+	n.Op.Format(w)
```

Comment:
for `create table t(a int as (+1))`,
MySQL:
```
CREATE TABLE `t` (
  `a` int(11) GENERATED ALWAYS AS (1) VIRTUAL
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```

for `create table t(a int as (-1))`
MySQL:
``` sql
CREATE TABLE `t` (
  `a` int(11) GENERATED ALWAYS AS (-(1)) VIRTUAL
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```

Reply:
I don't think here we need to be 100% with MySQL. we don't need to add extra `(` and `)` for neg operator.